### PR TITLE
get jobs by company

### LINKED ISSUE #74: List jobs by company
Currently, all jobs are being listed, but we only need to list jobs that were created by the company. Maybe should be created a new endpoint on the back-end to return a list of the Jobs by store. After creating this endpoint on the server-side, we need to update our request on the client-side.

### PR DESCRIPTION
**Pull request by edenilsonfe**
With this change we'll be able to get all jobs related to the logged company.
It closes #74

### CODE REVIEW

File: pages/admin/my-jobs.vue [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default {
   methods: {
     async getJobs() {
       this.loadingJobs = true
-      const jobs = await this.$jobRepository.index()
+      const jobs = await this.$companyRepository.getJobsByCompany()
```
Comment (carlosxjm) [inline pages/admin/my-jobs.vue]: Haven't it  pagination to this list?

File: plugins/axios.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ export default (ctx, inject) => {
 
   inject('eventRepository', repositoryWithAxios('/events'))
   inject('jobRepository', repositoryWithAxios('/job-offers'))
+  inject('companyRepository', repositoryWithAxios('/company'))
```
Comment (carlosxjm) [inline plugins/axios.js]: Use vuex strategy.

Comment (allanhal) [reply plugins/axios.js]: Let's do this in another iteration.

Comment (carlosxjm) [reply plugins/axios.js]: Ok

File: pages/admin/my-jobs.vue (hunk 1 above)
Comment (edenilsonfe) [reply pages/admin/my-jobs.vue]: I can do these changes to use pagination in another MR, after do this using Vuex strategy. What do you think about it?

**Pull request merged**